controller/src/main/java/io/pravega/controller/store/kvtable/InMemoryKVTMetadataStore.java

```diff
@@ -40,14 +39,9 @@
     @GuardedBy("$lock")
     private Map<String, InMemoryScope> scopes = new HashMap<>();
 
-    private final AtomicInt96 counter;
-
-    private final Executor executor;
 
     public InMemoryKVTMetadataStore(Executor executor) {
```

Comment:
Remove `executor` from the argument list.

Reply:
Ok. Sure.

Reply:
fixed